swap/tests/harness/mod.rs

```diff
@@ -420,8 +423,13 @@ impl BobParams {
         Ok((swap, event_loop))
     }
 
-    pub fn new_eventloop(&self, swap_id: Uuid) -> Result<(bob::EventLoop, bob::EventLoopHandle)> {
-        let mut swarm = swarm::bob(&self.seed, self.alice_peer_id)?;
+    pub async fn new_eventloop(
+        &self,
+        swap_id: Uuid,
+    ) -> Result<(bob::EventLoop, bob::EventLoopHandle)> {
+        let tor_socks5_port = get_port()
+            .expect("We don't care about Tor in the tests so we get a free port to disable it.");
```

Comment:
Why does it need to be a free port? Is `0` not sufficient?

Reply:
I did not try `0`. I did not want by accident chose a port which exists because we try to connect to this port and fail if it is not reachable.